### PR TITLE
EJB v0.4: links update 

### PR DESCRIPTION
**Pull request by bullet-tooth**

### CODE REVIEW

**Comment by dmitry-timofeev** [comment]
Please base on our branch with 0.4-related updates (beware, it might have new file-links): https://github.com/exonum/exonum-doc/tree/develop-ejb-0.4

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -480,9 +480,9 @@ For using the library just include the dependency in your `pom.xml`:
 - [Java Binding App tutorial][app-tutorial]
 
 [abstractservice]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/AbstractService.html
-[apicontrollertest]: https://github.com/exonum/exonum-java-binding/blob/v0.3/exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java
-[app-tutorial]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding-core/rust/ejb-app/TUTORIAL.md
-[build-description]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding-service-archetype/src/main/resources/archetype-resources/pom.xml
+[apicontrollertest]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java
+[app-tutorial]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding/core/rust/ejb-app/TUTORIAL.md
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: These links point to master, is that correct (tutorial and build-desc)?

File: src/get-started/java-binding.md [hunk 2]
```diff
@@ -492,7 +492,7 @@ For using the library just include the dependency in your `pom.xml`:
 [service]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/Service.html
 [standardserializers]: https://exonum.com/doc/api/java-binding-common/latest/com/exonum/binding/common/serialization/StandardSerializers.html
 [storage-indices]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/storage/indices/package-summary.html
-[submittransaction]: https://github.com/exonum/exonum-java-binding/blob/v0.3/exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ApiController.java
+[submittransaction]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/ApiController.java
```
Comment (dmitry-timofeev) [inline src/get-started/java-binding.md]: Would you double-check if this link is used, when rebase onto our 0.4 branch? Our demo doesn't seem to include such functionality as of 0.4.

**Comment by bullet-tooth** [comment]
Substituted by #271

**Pull request closed without merging**